docs/index.md

```diff
@@ -60,8 +60,10 @@ provider "zookeeper" {
 
 ### Optional
 
+- `password` (String, Sensitive) Password for digest authentication
 - `servers` (String) A comma separated list of 'host:port' pairs, pointing at ZooKeeper Server(s).
 - `session_timeout` (Number) How many seconds a session is considered valid after losing connectivity. More information about ZooKeeper sessions can be found [here](#zookeeper-sessions).
+- `username` (String, Sensitive) Username for digest authentication
```

Comment:
This part is generated by [tfplugindocs](https://github.com/hashicorp/terraform-plugin-docs). And looks like it just sorts alphabetically. Take a look at this example in the hashicorp's provider, it also has username and password separated: https://github.com/hashicorp/terraform-provider-tls/blob/main/docs/index.md#nested-schema-for-proxy

So I can change the order of the fields, but it will be overwritten each time we run `make generate`.

Reply:
Ah, silly me - of course.